cfgov/v1/util/migrations.py

```diff
@@ -77,17 +77,11 @@ def get_stream_data(page_or_revision, field_name):
     revision """
     if is_page(page_or_revision):
         field = getattr(page_or_revision, field_name)
-        stream_block = field.stream_block
-        stream_data = stream_block.get_prep_value(field)
+        return field.stream_data
```

Comment:
This is a nice simplification — I went back through the history to see if I could figure out why we weren't just using `field.stream_data` to begin with, and the [PR that last refactored this still](https://github.com/cfpb/cfgov-refresh/pull/2481) doesn't [yield any good reasons](https://github.com/cfpb/cfgov-refresh/pull/2481/commits/5a49cb65662ee03456bd72c08d9e95ad698062fa) on the commit that made the change.

Looks good to me!

Reply:
I agree, I like this change!

Reply:
[Right now our documentation says](https://cfpb.github.io/cfgov-refresh/migrations/#wagtail-specific-considerations) to have a schema migration to add the new fields, data migration to migrate, and then another schema migration to remove the old fields. Should the docs be updated with this change?

Reply:
I don't think this change impacts the need to do it that way, does it?

Reply:
It's complicated: in general best practice with data migrations is to always do them in multiple steps (because then you don't break the database out from under an older version of the code), although in practice we rarely do it that way. That documentation page could definitely be cleaned up; for example it mentions moving data "from the old field to the new field" but I think it means to talk about StreamField _blocks_.

One key thing about this change, is that it's more than a simplification but actually necessary for this to work. The issue is that if you load `stream_data` into Python, Wagtail strips out blocks that aren't currently defined in the model field. So if your code alters a StreamField to replace one block with another, and the data migration code uses our existing `get_stream_data`, you don't have a way to get the old block data to migrate it to the new block. Instead you have to use `.stream_data` to bypass the Python conversion.

Thanks for the reviews; at some point I'd like to revisit SF data migrations but wanted to do the minimum here to make this PR possible.